bbb-graphql-server/bbb_schema.sql

```diff
@@ -1298,6 +1298,7 @@ CREATE TABLE "pres_presentation" (
 	"downloadable" boolean,
 	"downloadFileExtension" varchar(25),
 	"downloadFileUri" varchar(500),
+    "renderedInToast" boolean,
```

Comment:
The name `renderedInToast` feels overly technical and doesn't intuitively communicate its purpose. From what I understand, this flag is meant to indicate whether a notification about this upload being complete has been shown. A clearer and more descriptive name could help make the code easier to read and maintain. For instance, I would suggest naming it `uploadCompletionNotified` or something similarly intuitive.